myhpi/core/models.py

```diff
@@ -79,12 +88,20 @@ class MinutesList(BasePage):
     ]
     subpage_types = ["Minutes"]
 
-    def get_context(self, request, *args, **kwargs):
-        context = super().get_context(request, *args, **kwargs)
+    def get_visible_minutes(self, request):
         minutes_ids = self.get_children().exact_type(Minutes).values_list("id", flat=True)
+        user_groups = get_user_groups(request.user)
         minutes_list = (
-            Minutes.objects.filter(id__in=minutes_ids, group=self.group).order_by("date").reverse()
+            Minutes.objects.filter(id__in=minutes_ids, group=self.group)
```

Comment:
we wanted to remove the group field from minutes (#98), you could also include that in this PR :)

Reply:
Done